akka-remote/src/test/scala/akka/remote/artery/RemoteDeathWatchSpec.scala

```diff
@@ -78,6 +87,10 @@ class RemoteDeathWatchSpec extends ArteryMultiNodeSpec(RemoteDeathWatchSpec.conf
   }
 
   "receive ActorIdentity(None) when identified node is unknown host" in {
+    // FIXME this is a difference between Aeron and TCP. AeronSink will throw exception
```

Comment:
Really? I'd expect the outbound connection to fail, which is wrapped in a restart so it'll restart a few times then fail. ArteryTcpTransport line 77-110

Reply:
I only mean that there is a timing difference. The end result is the same, it just takes longer for TCP. My thinking was that there might not be much meaning to do those inner restarts for UnknownHostException, but that is kind of hidden.

In the end it will log:

```
[ERROR] [01/24/2018 12:01:27.727] [RemoteDeathWatchSpec-akka.remote.default-remote-dispatcher-7] [akka.remote.artery.Association(akka://RemoteDeathWatchSpec)] Outbound message stream to [akka://RemoteDeathWatchSpec@unknownhost2:2552] failed. Restarting it. Tcp command [Connect(unknownhost2:2552,None,List(),Some(5 seconds),true)] failed because of unknownhost2 (akka.stream.StreamTcpException: Tcp command [Connect(unknownhost2:2552,None,List(),Some(5 seconds),true)] failed because of unknownhost2)
```